benchexec/tablegenerator/react-table/src/utils/utils.js

```diff
@@ -384,13 +398,14 @@ export const decodeFilter = (filterID) => {
   }
   const splitedArray = filterID.split("_");
 
-  if (splitedArray.length > 3) {
+  if (splitedArray.length === 2) {
     throw new Error("Invalid filter ID");
   }
   return {
     tool: splitedArray[0],
-    name: splitedArray[1],
-    column: splitedArray[2],
+    name:
+      splitedArray.length > 2 ? splitedArray.slice(1, -1).join("_") : undefined,
+    column: splitedArray.length > 2 ? splitedArray.at(-1) : undefined,
```

Comment:
`splitedArray.length > 2` should always be true, right (because of the check above)?

And it would be nice to have a comment that explains that `tool` is the first value, `column` the last, and `name` all values in between.

Reply:
The check above is just for `splitedArray.length === 2` . `filterID` has value `"id"` when id filters are applied, so `splitedArray` has length 1 and in this case, name & column should be `undefined` .  `splitedArray` will always have length equal to 1 or >= 3 .